Tasks/KubernetesManifestV0/src/actions/promote.ts

```diff
@@ -0,0 +1,22 @@
+"use strict";
+import tl = require('vsts-task-lib/task');
+import * as deploymentHelper from '../utils/DeploymentHelper';
+import * as canaryDeploymentHelper from '../utils/CanaryDeploymentHelper';
+import { Kubectl } from "kubernetes-common/kubectl-object-model";
+import * as utils from "../utils/utilities";
+import * as TaskInputParameters from '../models/TaskInputParameters';
+
+export async function promote() {
+
+    let kubectl = new Kubectl(await utils.getKubectl(), TaskInputParameters.namespace);
+
+    // Deploy input manifests
+    tl.debug("Deploying input manifests");
+    deploymentHelper.deploy(kubectl, TaskInputParameters.manifests, "None");
```

Comment:
This should be inside the if block for canary strategy check. If one uses action: promote and strategy: none, we should show an error stating that promote action works only with strategy: canary